ethtx/providers/web3_provider.py

```diff
@@ -33,43 +34,25 @@ def connect_chain(
     http_hook: str = None, ipc_hook: str = None, ws_hook: str = None, poa: bool = False
 ) -> Web3 or None:
     if http_hook:
-        method = "HTTP"
         provider = Web3.HTTPProvider
         hook = http_hook
     elif ipc_hook:
-        method = "IPC"
         provider = Web3.IPCProvider
         hook = ipc_hook
     elif ws_hook:
-        method = "Websocket"
         provider = Web3.WebsocketProvider
         hook = ws_hook
     else:
-        method = "IPC"
         provider = Web3.IPCProvider
         hook = "\\\\.\\pipe\\geth.ipc"
 
-    try:
-        w3 = Web3(provider(hook, request_kwargs={"timeout": 600}))
+    w3 = Web3(provider(hook, request_kwargs={"timeout": 600}))
```

Comment:
just saw this timeout, 600 is a lot, can we change it something way lower?

Reply:
That wasn't set up and calculated by you? 🧐